package.json

```diff
@@ -96,7 +96,10 @@
     "atLeast": 100,
     "detail": true,
     "ignoreCatch": true,
-    "strict": true
+    "strict": true,
+    "ignoreFiles": [
+      "test/*.js"
```

Comment:
Tests should also be types, it's essential for ensuring the library typing are correct.